### ISSUE TITLE
user avatar lost during following a user

### ISSUE THREAD

**Issue by utzer** [friendica/friendica#9715]
- [X] I have searched open and closed issues for duplicates
(but I am not sure if I missed it)

### Bug Description

When I follow a new contact the contact avatar/photo is not pulled, even more if it was there before on my instance it is lost when following a contact. 

### Steps to Reproduce

1. Look for some contact you do not follow yet. 
2. Follow this contact

Actual result:
When the process is complete the contact will show up with the dummy avatar. 
![image](https://user-images.githubusercontent.com/8361662/103210678-aada5980-4906-11eb-8bb0-35b6de6926fc.png)

Expected Result:
The contacts avatar should be pulled from the contact server in any case during the following of a contact. In the intermediate time the local avatar (if any) should already be used. 

### Platform Info

Friendica Version:
Current RC 12.2020

Friendica Source: 
git

PHP version:
PHP 7.4.13

SQL version:
mysql  Ver 15.1 Distrib 10.5.8-MariaDB

**Comment by annando** [comment]
Yeaj, I observed this behaviour for a longer period now. You can help yourself by refetching the contact information (in the menu on the contact page). We should solve this, but I don't think that it is blocking.

**Comment by utzer** [comment]
> Yeaj, I observed this behaviour for a longer period now. You can help yourself by refetching the contact information (in the menu on the contact page). We should solve this, but I don't think that it is blocking.

Sorry, should have mentioned the solution too. I do as you wrote every time. Took me some time to realize that this avatars are just not fetched.

**Comment by urbalazs** [comment]
I have a similar issue, but I'm not really sure it is connected to this one. I have a contact who posted some photos. I can see all the photos. When I go to the contacts page and see the details of this contact, then the photos (only the image files) won't be accessible anymore for me from this contact. The posts and the text in the post is accessible, but the photos are not. When I try to open the photo directly in the browser, I got this warning:

```
Not found

The Photo with id 86419518315fd33794b1b65047314915 is not available.

#0 [internal function]: Friendica\Module\Photo::init()
#1 /var/www/friendica-full-2020.09-1/src/App/Module.php(271): call_user_func()
#2 /var/www/friendica-full-2020.09-1/src/App.php(555): Friendica\App\Module->run()
#3 /var/www/friendica-full-2020.09-1/index.php(42): Friendica\App->runFrontend()
#4 {main}
```

Re-fetching the contact information doesn't solve the issue.

**Comment by annando** [comment]
@urbalazs this is unrelated. Please open a different issue. This here is about adding a contact.

**Comment by annando** [comment]
Fixed by #10465
